test/area-stack.html

```diff
@@ -576,9 +572,186 @@
 
         </script>
 
+        <script>
+
+            var chart;
+            var option;
+
+            require(['echarts'], function (echarts) {
+
+                var             option = {
```

Comment:
Extra space here.

Reply:
@pissang uhh, sorry for not catching this 🤦  This is now fixed, thanks for the review!